learn/about.mdx

```diff
@@ -0,0 +1,34 @@
+import Card from "react-bootstrap/Card";
+import CardGroup from "react-bootstrap/CardGroup";
+import Link from "@docusaurus/Link";
+
+# About
+
+Who are we and where can you find us?
+
+## Bots
+
+<CardGroup style={{ width: "30rem" }}>
```

Comment:
Are adding bootstrap for this only? Not sure if the extra dependency + CSS is worth it when this is essentially a table. We'd need to assess impact on load times and data transfers.